qa-test-app/src/main/java/com/mapbox/navigation/qa_test_app/view/MapboxNavigationViewCustomizedActivity.kt

```diff
@@ -69,7 +69,7 @@ class MapboxNavigationViewCustomizedActivity : AppCompatActivity() {
                     )
                     .build()
             )
-            .withRouteLineBelowLayerId("road-label-navigation")
+            .withRouteLineBelowLayerId("road-label") // for Style.LIGHT and Style.DARK
```

Comment:
@abhishek1508 I think its a good idea to mention in Drop-In UI documentation that when overriding default Navigation Map Style, developers should also override `routeLineOptions` with `.withRouteLineBelowLayerId` value that points at the layer that exists in their Style.

Reply:
This is already mentioned [here](https://docs.mapbox.com/android/navigation/guides/ui-components/route-line/#position-the-route-line-relative-to-map-layers)